addon/mixins/regenerated/serializers/new-platform-flexberry-g-i-s-layer-metadata.js

```diff
@@ -4,14 +4,18 @@
 
 import Ember from 'ember';
 
-/**
-  Mixin containing map layer metadata model serializer properties.
-
-  @class NewPlatformFlexberyGISLayerMetadataSerializerMixin
-  @extends <a href="http://emberjs.com/api/classes/Ember.Mixin.html">Ember.Mixin</a>
-*/
```

Comment:
?

Reply:
Result of regeneration. I'll try to restore them.
But each regenaretion will remove comments again.